### PR TITLE
Adding l5kit and dataset installation at the top of the notebook

### PR DESCRIPTION
**Pull request by perone**
This PR will add a hidden cell (on google colab) to automatically install the sample dataset as well as l5kit. This enables the notebook to run without any problems in Google colab.

The cell looks like this:
image

### CODE REVIEW

File: examples/visualisation/visualise_data.ipynb
Comment (lucabergamini) [inline examples/visualisation/visualise_data.ipynb:61]: don't we need also to move  `meta.json` into the root folder ? It should be inside `semantic_map.tar`

File: examples/visualisation/visualise_data.ipynb
Comment (lucabergamini) [inline examples/visualisation/visualise_data.ipynb:68]: `pyyaml` should already been pulled from `l5kit` I think..

**Review by lucabergamini — COMMENTED** [review]
minors, I'll give it a try later!

File: examples/visualisation/visualise_data.ipynb
Comment (perone) [reply examples/visualisation/visualise_data.ipynb:68]: I think this is an issue w/ Colab, because it required to have this last version work there.

File: examples/visualisation/visualise_data.ipynb
Comment (perone) [reply examples/visualisation/visualise_data.ipynb:61]: The semantic map is already being decompressed directly into the destination folder. Is that what you mean ?

Comment (lucabergamini) [reply examples/visualisation/visualise_data.ipynb:61]: isn't it decompress inside `l5kit_dataset/semantic_map` ?

Comment (perone) [reply examples/visualisation/visualise_data.ipynb:61]: Yes, everything from the `semantic_map.tar` will go into `l5kit_dataset/semantic_map`.

Comment (lucabergamini) [reply examples/visualisation/visualise_data.ipynb:61]: ah I see what's happening then!

I think we're falling back to [this](https://github.com/lyft/l5kit/blob/19b7ac8a35f50467ee8916b58e60298b5f5e7146/l5kit/l5kit/rasterization/rasterizer_builder.py#L144) or [this](https://github.com/lyft/l5kit/blob/19b7ac8a35f50467ee8916b58e60298b5f5e7146/l5kit/l5kit/rasterization/rasterizer_builder.py#L161) then

I've checked and the `meta.json` is not where it should be in fact...this could become an issue once we remove the hard-coded matrices

File: examples/visualisation/visualise_data.ipynb
Comment (lucabergamini) [inline examples/visualisation/visualise_data.ipynb:44]: any chance we can get a tmp dir here?

Comment (perone) [reply examples/visualisation/visualise_data.ipynb:44]: Let's have a sync about that because since there are other notebooks I was thinking of moving this functionality to a Python code inside of the l5kit module.

File: examples/visualisation/visualise_data.ipynb
Comment (perone) [reply examples/visualisation/visualise_data.ipynb:61]: Wasn't aware of these matrices there, I think we should add another PR adding warnings upon fallback, I will open a new issue for us to keep track.

Comment (lucabergamini) [reply examples/visualisation/visualise_data.ipynb:61]: ideally we should remove the fallback entirely. The dataset has been updated now

**Comment by perone** [comment]
@lucabergamini PTAL, I moved the script to a script file and added the temp dir. The code in the top of the notebook to setup for google colab will only run on google colab as well. To test it, you will have to change the link to download the script to [this one](https://raw.githubusercontent.com/lyft/l5kit/colab_notebook_viz/examples/setup_notebook_colab.sh), because it is pointing to the one after merge of this PR (master).

**Comment by perone** [comment]
@lucabergamini I want to merge this soon because the viz notebook is failing on colab.

File: examples/setup_notebook_colab.sh
Comment (lucabergamini) [inline examples/setup_notebook_colab.sh:20]: can we move `meta.json` in `TEMP_DATASET_DIR`?

**Review by lucabergamini — DISMISSED** [review]
minor, but LGTM

File: examples/setup_notebook_colab.sh
Comment (perone) [reply examples/setup_notebook_colab.sh:20]: I totally forgot about it, will fix it.

Comment (perone) [reply examples/setup_notebook_colab.sh:20]: Fixed

**Comment by perone** [comment]
@lucabergamini I added the copy of the `meta.json`, it will require another review to be able to merge it.

**Review by lucabergamini — APPROVED** [review]
LGTM

**Pull request merged**
